frontend/src/components/Body/BodyThree/BodyThree.css

```diff
@@ -0,0 +1,38 @@
+.BodyThree {
+    align-content: center;
+    background-color: rgb(254, 239, 221);
+    display: flex;
+    font-family: 'Gill Sans', sans-serif;
+    margin: auto;
+}
+
+.Text-container-B3 {
+    justify-content: flex-start;
+    background-color: rgb(254, 239, 221);
+    display: grid;
+    white-space: normal;
+    margin-left: 70px;
+}
+
+.Text-container-B3 h2 {
+    color: rgb(255, 64, 0);
+    text-align: start;
+    font-size: 2em;
+}
+
+.Text-container-B3 p {
+    margin: 0;
+    font-size: 0.8em;
+    color: rgb(32, 30, 31);
+}
+
+.IMG-container-B3 {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    padding: 50px;
+}
+
+.IMG-container-B3 img {
+    display: block;
+}
```

Comment:
it feels silly but let's make sure we add a new line/empty line at the end of each file, which will get rid of these red icons